fossunited/fossunited/doctype/foss_project_grant/foss_project_grant.py

```
@@ -18,6 +18,8 @@ class FOSSProjectGrant(Document):
         date_of_provision: DF.Date | None
         grant_amount: DF.Data | None
         grant_status: DF.Literal["Open", "Approved", "Under Review", "Rejected"]  # noqa: F722, F821
+        grant_type: DF.Literal["Project", "Fellowship"]  # noqa: F722, F821
```

Ruff (0.14.1)

Unused `noqa` directive (unused: `F722`)

Remove unused `noqa` directive

(RUF100)